homeassistant/components/mqtt/client.py

```diff
@@ -100,6 +101,9 @@
 TIMEOUT_ACK = 10
 RECONNECT_INTERVAL_SECONDS = 10
 
+MAX_SUBSCRIBES_PER_CALL = 100
+MAX_UNSUBSCRIBES_PER_CALL = 100
```

Comment:
We currently do upwards of 5000 in some cases which we know is too big.  I think 100 might generate more than we want so I was thinking 500 here.